main/src/cgeo/geocaching/log/LogsViewCreator.java

```diff
@@ -96,13 +107,13 @@ protected void fillViewHolder(@SuppressWarnings("unused") final View convertView
         if (log.hasLogImages()) {
             holder.images.setText(log.getImageTitles());
             holder.images.setVisibility(View.VISIBLE);
-            holder.images.setOnClickListener(v -> ImagesActivity.startActivity(activity, getGeocode(), new ArrayList<>(log.getLogImages())));
+            holder.images.setOnClickListener(v -> ImagesActivity.startActivity(activity, getGeocode(), new ArrayList<>(log.logImages)));
```

Comment:
Just wonder, why do you discard usage of method and use member instead?

Reply:
member function was deprecated

Reply:
Okay, I'm not really following all the code changes, but typically it's a good practice to use methods over member variables... Anyway, no problem here

Reply:
@okainov there is (or maybe was? I remember this from a couple of years ago...) a big debate regarding how to model immutable value objects in Java. While one side argues that also for immutable objects (which of course have no setter methods) explicit getter objects shall be used, the other argues that in this case one can also access directly final members. If you google "java immutable objects" you will find lots of sides about this.

I personally don't mind which pattern is used :-) BUt whoever modelled LogEntry in the first place was clearly advocating the second solution. So I decided to follow it. Because although I don't care too much which pattern to follow (as long as it is sensible enough), I like it when all code follows the SAME pattern :-)